test_conformance/c11_atomics/common.h

```diff
@@ -873,20 +872,23 @@ std::string CBasicTest<HostAtomicType, HostDataType>::KernelCode(cl_uint maxNumD
         "  if(get_local_id(0) == 0) // first thread in workgroup\n";
     else
       // global atomics declared in program scope
-      code +=
-      "  if(atomic_fetch_add(&finishedThreads, 1) == get_global_size(0)-1)\n"
-      "    // last finished thread\n";
+      code += "  if(atomic_fetch_add_explicit(&finishedThreads, 1) == "
```

Comment:
I think the line got a bit muddled, the ordering and scope arguments to the atomic aren't being passed in.

Reply:
Thanks for such quick feedback! Yes, good catch, I guess we were never actually hitting this case, have fixed.